apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx

```diff
@@ -147,7 +147,6 @@ export const ReceiptsTable: React.FC<IContextProps> = ({ context }) => {
             <ServicesModal
                 receipt={detailedReceipt}
                 visible={modalIsVisible}
-                onOk={hideServiceModal}
```

Comment:
Did it change some logic there? Or there was no ok button before?

Reply:
There was the same logic as with `onCancel`.
But this modal window already had `footer: null`, so the OK button was not there